src/mirror/services.md

```diff
@@ -0,0 +1,355 @@
+
+  ## Default flushing interval for all outputs. Maximum flush_interval will be
+  ## flush_interval + flush_jitter
+  flush_interval = "10s"
+  ## Jitter the flush interval by a random amount. This is primarily to avoid
+  ## large write spikes for users running a large number of telegraf instances.
+  ## ie, a jitter of 5s and interval 10s means flushes will happen every 10-15s
+  flush_jitter = "0s"
+
+  ## By default or when set to "0s", precision will be set to the same
+  ## timestamp order as the collection interval, with the maximum being 1s.
+  ##   ie, when interval = "10s", precision will be "1s"
+  ##       when interval = "250ms", precision will be "1ms"
+  ## Precision will NOT be used for service inputs. It is up to each individual
+  ## service input to set the timestamp at the appropriate precision.
+  ## Valid time units are "ns", "us" (or "µs"), "ms", "s".
+  precision = ""
+
+  ## Override default hostname, if empty use os.Hostname()
+  hostname = ""
+
+  ## If set to true, do no set the "host" tag in the telegraf agent.
+  omit_hostname = false
+[[outputs.influxdb_v2]]
+  ## The URLs of the InfluxDB cluster nodes.
+  urls = ["https://localhost:8086"]
+
+  ## Token for authentication.
+  token = "REDACTED"
+
+  ## Organization is the name of the organization you wish to write to; must exist.
+  organization = "COSI"
+
+  ## Destination bucket to write into.
+  bucket = "system"
+
+  ## Optional TLS Config for use on HTTP connections.
+  # tls_ca = "/etc/telegraf/ca.pem"
+  # tls_cert = "/etc/telegraf/cert.pem"
+  # tls_key = "/etc/telegraf/key.pem"
+
+  ## Use TLS but skip chain & host verification
+  insecure_skip_verify = true
+[[inputs.cpu]]
+  ## Whether to report per-cpu stats or not
+  percpu = true
+
+  ## Whether to report total system cpu stats or not
+  totalcpu = true
+
+  ## If true, collect raw CPU time metrics
+  collect_cpu_time = false
+
+  ## If true, compute and report the sum of all non-idle CPU states
+  report_active = false
+[[inputs.disk]]
+  ## Ignore mount points by filesystem type.
+  ignore_fs = ["tmpfs", "devtmpfs", "devfs", "iso9660", "overlay", "aufs", "squashfs"]
+[[inputs.diskio]]
+[[inputs.mem]]
+[[inputs.net]]
+[[inputs.nginx]]
+  # An array of Nginx stub_status URI to gather stats.
+  urls = ["http://localhost/status"]
+
+  # HTTP response timeout (default: 5s)
+  response_timeout = "5s"
+[[inputs.system]]
+[[inputs.zfs]]
+```
+
+## RKHunter
+
+_updated: Nov 6th 2022_
+
+[RKHunter](https://rkhunter.sourceforge.net/) or "Rootkit Hunter" is a script that checks linux systems for signs of known linux rootkits. Assuming other security measures are well followed we should not expect Mirror to become infected with a rootkit. It might be worth an investment in running the tool automaticaaly (maybe daily?) but we don't currently have satisfactory methods of sending mail when warnings occur. Until then (and maybe forever) here is some instructions on manually running RKHunter. 
```

Comment:
```suggestion
[RKHunter](https://rkhunter.sourceforge.net/) or "Rootkit Hunter" is a script that checks linux systems for signs of known linux rootkits. Assuming other security measures are well followed we don't expect Mirror to become infected with a rootkit. It might be worth running the tool automatically (maybe daily?), however, we don't currently have a satisfactory method of sending mail when warnings occur. Until then (and maybe forever) here are some instructions for manually running RKHunter. 
```